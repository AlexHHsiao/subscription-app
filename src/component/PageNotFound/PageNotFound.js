import React from 'react';
import './PageNotFound.scss';

const PageNotFound = () => (
    <div className='notFoundContainer'>
        <div className='notFoundContent'>
            <h4>The page you are looking does not exist!</h4>
        </div>
    </div>
);

export default PageNotFound;
