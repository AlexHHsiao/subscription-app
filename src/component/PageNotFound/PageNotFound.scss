.notFoundContainer {
  display: block;
  position: fixed;
  left: 0;
  top: 0;
  width: 100vw;
  height: 100vh;
}

.notFoundContent {
  display: block;
  position: absolute;
  width: 100%;
  height: 200px;
  top: 0;
  bottom: 0;
  margin: auto;
  text-align: center;
}
