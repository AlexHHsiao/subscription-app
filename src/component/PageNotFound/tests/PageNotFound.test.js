import React from 'react';
import {render} from '@testing-library/react';
import PageNotFound from '../PageNotFound';

test('<PageNotFound/>', () => {
    const {container: {firstChild}} = render(<PageNotFound/>);
    expect(firstChild).toMatchSnapshot();
});
