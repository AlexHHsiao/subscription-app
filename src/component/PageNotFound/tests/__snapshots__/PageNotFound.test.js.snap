// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageNotFound/> 1`] = `
<div
  class="notFoundContainer"
>
  <div
    class="notFoundContent"
  >
    <h4>
      The page you are looking does not exist!
    </h4>
  </div>
</div>
`;
